nixos/modules/services/networking/xrdp.nix

```diff
@@ -91,6 +91,15 @@ in
         '';
       };
 
+      logFile = mkOption {
+        type = types.str;
+        default = "/dev/null";
+        example = "/var/log/xrdp-sesman.log";
+        description = ''
+          location for xrdp to write its logs
```

Comment:
could be something more like "Location where xrdp writes its logs." with capitals/period but